modules/aaudio/aaudio.h

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file aaudio/aaudio.h AAudio audio driver for Android
+ *
+ * Copyright (C) 2024 Juha Heinanen
+ */
+
+#include <aaudio/AAudio.h>
+
+void close_stream(AAudioStream *stream);
```

Comment:
Please use the name of the module as prefix for global symbols ...

Reply:
Done by https://github.com/baresip/baresip/commit/f5c2e3338770422c2421782aeb12391392f9b391